scripts/managedb/mdsvector2mdsobject.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# -*- coding: latin-1 -*-
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+
+# This script is inspired in old attrsvector2attrsobject.py
+
+__author__ = 'fermin'
+
+from pymongo import MongoClient
+import json
+import sys
+from time import sleep
+
+ATTRS     = 'attrs'
+MDS       = 'md'
+MD_NAMES  = 'mdNames'
+
+# The way in which Python manage dictionaries doesn't make easy to be sure
+# of field ordering, which is important for MongoDB in the case of using an
+# embedded document for _id. The flatten() function helps
```

Comment:
Not sure... I tend to prefer "help" as the solution provided by the function is not as smart as I'd like.

NTC